activestorage/db/migrate/20170806125915_create_active_storage_tables.rb

```diff
@@ -3,7 +3,7 @@ def change
     # Use Active Record's configured type for primary and foreign keys
     primary_key_type, foreign_key_type = primary_and_foreign_key_types
 
-    create_table :active_storage_blobs, id: primary_key_type do |t|
+    create_table ActiveStorage::Blob.table_name, id: primary_key_type do |t|
```

Comment:
Per my understanding, this will break the migration if ActiveStorage is not available anymore (like removed later from Gemfile). Just saying, I'm not sure that's worth to handle.

Reply:
Thanks for the hint. This could be a problem for a rollback or down migration. But if ActiveStorage is removed from a project, you will probably have to adapt the database schema manually anyway. I see much more of a need for a migration that renames these tables in existing projects from plural to singular, if the `config.active_record.pluralize_table_names` is set accordingly.

Reply:
Or what if you have `pluralize_table_names` off when you install ASto but turn it back on later?

These migrations are designed to get you started in the most common case, if you're changing the core behavior of table naming that is on you I think.

Reply:
Yeah, sorry I just think that setting is expected to have side-effects and we should be aware of them -- changing this only for that reason seems too risky. Let's document the behavior for now unless there are other reasons to do it.

Reply:
To problem. I have way too little experience to be aware of the consequences of such a change.
What would be a proper way to document the behavior? Should I create an issue for this? Or is there a suitable place in the documentation?

Reply:
I think it could be documented in the configuring guide:
https://github.com/rails/rails/blob/5465c7de22aa56f86d7446152166837de8092c0a/guides/source/configuring.md?plain=1#L1060-L1062

As well as in the API docs for `pluralize_table_names=`:
https://github.com/rails/rails/blob/5465c7de22aa56f86d7446152166837de8092c0a/activerecord/lib/active_record/model_schema.rb#L104-L110